slate-react/build.boot

```diff
@@ -0,0 +1,32 @@
+(def +lib-version+ "0.10.17")
+(def +version+ (str +lib-version+ "-0"))
+
+(set-env!
+ :resource-paths #{"resources"}
+ :dependencies '[[cljsjs/boot-cljsjs "0.8.2" :scope "test"]])
```

Comment:
You should add dependency on `cljsjs/react`,  `cljsjs/react-dom` and `cljsjs/slate` here.